src/main/java/org/ow2/proactive/catalog/service/CatalogObjectService.java

```diff
@@ -181,7 +182,7 @@ public CatalogObjectMetadata createCatalogObject(String bucketName, String name,
 
         BucketEntity bucketEntity = findBucketByNameAndCheck(bucketName);
 
-        CatalogObjectRevisionEntity catalogObjectEntityCheck = catalogObjectRevisionRepository.findDefaultCatalogObjectByNameInBucket(bucketName,
+        CatalogObjectRevisionEntity catalogObjectEntityCheck = catalogObjectRevisionRepository.findDefaultCatalogObjectByNameInBucket(Arrays.asList(bucketName),
```

Comment:
The called method should not do any changes to your structure? So `Collections.singletonList` might be more appropriate here

Reply:
yes, no change to the structure. why?

Reply:
changed to Collections.singletonList